src/structures/SkyBlock/SkyblockMember.js

```diff
@@ -401,7 +401,11 @@ class SkyblockMember {
  */
 /**
  * @typedef {object} SkyblockMemberDungeonsTypes
- * @property {SkyblockSkillLevel} catacombs
+ * @property {object} catacombs
+ * @property {SkyblockSkillLevel} catacombs.experience
```

Comment:
`catacombs.` wont work with jsdoc and will just show `catacombs.<what ever was inputted>` and not make show an new object